admin_guide/high_availability.adoc

```diff
@@ -252,23 +266,85 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
-. Depending on your environment policies, you can either reuse the *router*
-service account created previously or create a new *ipfailover* service account.
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
+
+|--watch-port
+|`*OPENSHIFT_HA_MONITOR_PORT*`
+|80
+|The ipfailover pod will test that it can open a TCP connection to this port. If it can, then the service is considered to be running.
+
+|--interface
+|`*OPENSHIFT_HA_NETWORK_INTERFACE*`
+|
+|The interface name for the ip failover to use to send VRRP traffic. By default, all interfaces are used.
+
+|
+|`*OPENSHIFT_HA_REPLICA_COUNT*`
+|2
+|This must match spec.replicas in the DC
+
+|--virtual-ips
+|`*OPENSHIFT_HA_VIRTUAL_IPS*`
+|
+|The list of IP address ranges to replicate. This must be provided. E.g., 1.2.3.4-6,1.2.3.9
+
+|--vrrp-id-offset
+|`*OPENSHIFT_HA_VRRP_ID_OFFSET*`
+|0
+|See xref:../admin_guide/high_availability.adoc#ha-vrrp-id-offset[VRRP Id Offset] discussion.
+
+|--iptables-chain
+|`*OPENSHIFT_HA_IPTABLES_CHAIN*`
+|INPUT
+|The name of the iptables chain to automatically add an iptables rule to allow the VRRP traffic on. If the value is not set, an iptables rule will not be added. If the chain does not exist it is not created.
+
+|===
+
+[[ha-vrrp-id-offset]]
+=== VRRP Id Offset
+Every VIP that is serviced by keepalived must have a unique VRRP Id Offset. Each ipfailover DC specifies
+a list of 1 or more VIPs that it manages.  By default when the first ipfailover DC is created the first
+VIP in the VIP list is assigned a value of 0.  The next gets the value 1, and so forth. When there are
+multiple ipfailover DCs the `--vrrp-id-offset=<n>` option is used to specify the value of the first VRRP
+Id Offset of the first VIP in the list of configured VIPs. It is up to the admin to make sure the lists
+do not collide with each other. VRRP Id is in the range 0..255.
```

Comment:
Aah, this makes is sound like the vrrp-id offset is on the DC. Its really the id of the keepalived peers. 

As an example: if you create an ipfailover config at an offset of 0 and have a replica count of 4 (`--replicas=4`) and there's a 4 nodes/pods that run keepalived, then the ids `0-3` would be used up. 
The next/second ipfailover configuration should use a vrrp-id offset of atleast 4. The id offsets exist to ensure that there is collision across groups (so replica count is important as well here).  Should mention that - this would need some rework.